src/budy/models/product.py

```diff
@@ -294,6 +300,18 @@ def simple_csv_url(cls, absolute = False):
             absolute = absolute
         )
 
+    @appier.operation(
+        name = "Apply Collection",
+        parameters = (
+            ("Collection", "collection", "str"),
+        )
+    )
+    def apply_collection(self, collection):

```

Comment:
This is failing saying that method is expecting two arguments but only one is provided. I found no examples of non class method operations that accepted arguments in the entire workspace.